WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -162,7 +162,11 @@ private extension HubMenu {
             case HubMenuViewModel.Customers.id:
                 CustomersListView(viewModel: .init(siteID: viewModel.siteID))
             case HubMenuViewModel.PointOfSaleEntryPoint.id:
-                PointOfSaleEntryPointView(currencySettings: ServiceLocator.currencySettings, hideAppTabBar: { isHidden in
+                // TODO:
+                // PointOfSaleEntryPointView does not need currencySettings, since these are passed through POSItemProvider
```

Comment:
nit: just checking, can we already remove currencySettings here, or is it still needed for other POS use cases and we plan to remove it in the future?

Reply:
I believe it can be removed, I've raised the same question here: https://github.com/woocommerce/woocommerce-ios/pull/12882#discussion_r1621778958 and will tackle it on https://github.com/woocommerce/woocommerce-ios/issues/12918 just to keep things separate 👍